src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -53,15 +56,26 @@ const ImagePreviewContainer = styled.View`
   marginBottom: 10
 `
 
+const PDFPreviewContainer = styled.View`
+  marginBottom: 10
+`
+
 interface Props extends RelayProps {
   senderName: string
   artworkPreview?: JSX.Element
-  imagePreview?: JSX.Element
 }
 
 export class Message extends React.Component<Props, any> {
   render() {
     const date = moment(this.props.message.created_at).fromNow(true)
```

Comment:
I’m seeing this pattern a lot in Emission’s code-base. Doesn’t MP have support out of the box for date formatting?